### PR TITLE
20220727 updates corrections

### PR DESCRIPTION
**Pull request by wkande**
Minor corrections and updates

### CODE REVIEW

File: docs/dao-members/introduction/api3-dao.md [hunk 1]
```diff
@@ -56,8 +57,8 @@ based on expert opinion. As API3 operations expand, this governance hierarchy
 may demand additional layers in the form of subDAOs.
 
 To learn more about hierarchical team structures
-<a href="/api3-whitepaper-v1.0.3.pdf#API3%20DAO" target="_api3-whitepaper"> see
-Section 5.3 of the API3 Whitepaper</a>.
+<a href="/api3-whitepaper-v1.0.3.pdf#page=21" target="_blank"> see Section 5.3
```
Comment (dcroote) [inline docs/dao-members/introduction/api3-dao.md:60]:
```suggestion
<a href="/api3-whitepaper-v1.0.3.pdf#page=22" target="_blank"> see Section 5.3
```

**Review by dcroote — COMMENTED** [review]
Nice, only one minor page update.

File: docs/dao-members/introduction/api3-dao.md (hunk 1 above)
Comment (wkande) [reply docs/dao-members/introduction/api3-dao.md:60]: @dcroote - I made this change in PR #945 s part of today's corrections.

Comment (dcroote) [reply docs/dao-members/introduction/api3-dao.md:60]: Nice, thanks for letting me know 👍

**Pull request merged**